docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -140,7 +140,7 @@ Note that the information reported as Pod status depends on the current
 A PodSpec has a `restartPolicy` field with possible values Always, OnFailure,
 and Never. The default value is Always.
 `restartPolicy` applies to all Containers in the Pod. `restartPolicy` only
-refers to restarts of the Containers by the kubelet on the same node. Failed
```

Comment:
based on the consensus feedback, please use the term "Exited" rather than "Terminated"